workflows/edit_data_packages/update_a_package.Rmd

```diff
@@ -18,7 +18,7 @@ pkg <- get_package(adc_test, "resource_map_pid")
 update <- publish_update(adc_test, 
                          metadata_pid = pkg$metadata,
                          resource_map_pid = pkg$resource_map,
-                         data_pids = c(pkg$data, id_new),
+                         data_pids = c(pkg$data, id_new), # delete existing pid to be updated and add new pid
```

Comment:
this doesn't delete the existing pid in this case. It adds a data pid... so "adding a datafile"

if you did this it will delete the existing pid(s)
`data_pids = id_new`

so it depends on what you are trying to accomplish and how many data pids you have